build/Dockerfile

```diff
@@ -1,11 +1,29 @@
-FROM golang:1.14 as builder
+# syntax = docker/dockerfile:experimental
```

Comment:
Is this parsed by something?

Reply:
Dockerfile used this to use new features

Reply:
not need for current build. i removed this.